ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-ref.yaml

```diff
@@ -52,6 +52,10 @@ ref:
       The instance type to use for compute nodes (e.g. GCP https://cloud.google.com/compute/docs/machine-types). We use a 4 core worker to match the median configuration of the fleet.
       Do not change the default family type without consulting with cloud financial operations as their may
       be active savings plans targeting this machine class.
+  - name: CONTROL_PLANE_NODE_TYPE
```

Comment:
@gpei We have step `ipi-conf-gcp-control-plane-node-type` to set the control-plane node type, shall we insert that step in the workflow instead?

Reply:
@jianli-wei good to know this, I thought we don't have parameter to customize this, let me use the existing one instead